mllib/src/main/scala/org/apache/spark/ml/clustering/GaussianMixture.scala

```diff
@@ -104,6 +105,25 @@ class GaussianMixtureModel private[ml] (
   @Since("2.0.0")
   def gaussians: Array[MultivariateGaussian] = parentModel.gaussians
 
+  /**
+   * Helper method used in Python.
```

Comment:
Remove this 1 line.  (This is an implementation detail and should not be exposed in user docs.)